.github/workflows/build-pullrequest.yml

```diff
@@ -28,7 +28,15 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
-
+      - name: Apply formatter
+        run: ./gradlew spotlessApply
+      - name: Commit formatting changes
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add .
+          git commit -m "Apply formatter" -a
+        continue-on-error: true
```

Comment:
This will still show if there is an error in spotless, but will let the build finish, right? Hopefully spotless won't encounter errors like that... I'm a little worried about cases where spotless might produce output that it itself considers invalid, so each time it runs it changes some section more and more. We've already noticed some oddities in indentation.

Reply:
That's a misunderstanding. The `continue-on-error` is only there because making a commit without changes, aka correct formatted files, would let the job fail because of exit code 1